src/main/java/ch/njol/skript/expressions/ExprBeaconValues.java

```diff
@@ -0,0 +1,199 @@
+package ch.njol.skript.expressions;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.classes.Changer.ChangeMode;
+import ch.njol.skript.doc.*;
+import ch.njol.skript.expressions.base.PropertyExpression;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.ExpressionType;
+import ch.njol.skript.lang.SkriptParser;
+import ch.njol.skript.util.Getter;
+import ch.njol.util.Kleenean;
+import ch.njol.util.coll.CollectionUtils;
+import org.bukkit.block.Beacon;
+import org.bukkit.event.Event;
+import org.bukkit.potion.PotionEffectType;
+import org.jetbrains.annotations.Nullable;
+import org.jetbrains.annotations.UnknownNullability;
+
+@Name("Beacon Values")
+@Description({
+	"Values of a Beacon",
+	"Secondary effect can only be set to regeneration.",
+	"Secondary effect can only be set when beacon tier is maxxed.",
+	"You can only change the range on Paper.",
+	"You can only set primary and secondary effects to null (clear) on Paper."
```

Comment:
```suggestion
	"The active effects of the beacon.",
	"The secondary effect can only be set to regeneration and it can only be set when the beacon has achieved the highest tier.",
	"You can only change the range and clear the primary and secondary effects on Paper."
```